pages/agent/v3/help/_env_set.md

```diff
@@ -21,27 +21,26 @@ Sets environment variables in the current job execution environment.
 Changes to the job environment variables only apply to subsequent phases of the job.
 This command cannot unset Buildkite read-only variables.
 
-To read the new values of variables from within the current phase, use ′env get′.
+To read the new values of variables from within the current phase, use `env get`.
 
 Note that this subcommand is only available from within the job executor with the job-api experiment enabled.
 
 ### Examples
-
```

Comment:
Nitpick: Should keep a blank line after headings.

Reply:
It's back now